packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -176,22 +176,18 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
 
   async fetchInitialData(): Promise<void> {
-    try {
+    // If get properties fails we dont want to try to get the participants after.
+    await this.asyncTeeErrorToEventEmitter(async () => {
       await this.chatThreadClient.getProperties();
```

Comment:
After thinking about this a bit, I believe we could merge 2 async () functions together, so it will be:

```
 await this.asyncTeeErrorToEventEmitter(async () => {
      await this.chatThreadClient.getProperties();
      for await...
});
```

Reply:
I was thinking about this as well since if the first function fails the error will surface. something I am wondering though (will test and report back) is does the emitter support calling 2 functions inside it?

Reply:
Emitter doesn't care about what you do in the closure. It just pipes the thrown exception, if any.

Reply:
Thanks for the clarification will update